.github/workflows/phar-creation.yml

```diff
@@ -4,22 +4,20 @@ name: "Building Require-Checker phar"
 
 on:
     pull_request:
+#    release:
+#      types: [published]
 
 jobs:
     build:
         name: "Building Require-Checker phar"
 
-        runs-on: ${{ matrix.operating-system }}
+        runs-on: "ubuntu-latest"
 
         strategy:
             matrix:
-                dependencies:
-                    - "locked"
                 php-version:
                     - "7.4"
```

Comment:
PHAR release should probably be done with only one version of PHP (lowest supported version, I'd say)

Reply:
it's only running for 7.4 now